griptape/events/start_structure_run_event.py

```diff
@@ -1,7 +1,15 @@
-from attrs import define
+from typing import Optional, Union
+from collections.abc import Sequence
+
+from attrs import define, field
+
+from griptape.artifacts import BaseArtifact
 from griptape.events.base_event import BaseEvent
 
 
 @define
 class StartStructureRunEvent(BaseEvent):
-    ...
+    input_task_input: Union[
+        BaseArtifact, tuple[BaseArtifact, ...], tuple[BaseArtifact, Sequence[BaseArtifact]]
+    ] = field(kw_only=True, metadata={"serializable": True})
```

Comment:
Not for this PR, but maybe we should define all these artifact output options as a union type?